huxley/www/static/js/huxley/components/AdvisorProfileView.js

```diff
@@ -243,9 +324,105 @@ var AdvisorProfileView = React.createClass({
           </div>
           <div className="tablemenu footer">
           </div>
+          <Button
+            color="green"
+            loading={this.state.loading}
+            type="submit">
+            Save
+          </Button>
+          <span className="help-text"><em> Remember to save any changes!</em></span>
         </form>
       </InnerView>
     );
   },
+
+  renderError: function(field) {
+    if (this.state.errors[field]) {
+      return (
+        <label className="hint error">
+          {this.state.errors[field]}
+        </label>
+      );
+    }
+
+    if (this.state.errors.school &&
+        this.state.errors.school[field]) {
+      return (
+        <label className="hint error">
+          {this.state.errors.school[field]}
+        </label>
+      );
+    }
+
+    return null;
+  },
+
+  _handleChange: function(fieldName, event) {
```

Comment:
Might as well put this in its own file so we can import it into other components.